src/Bundle.ts

```diff
@@ -261,6 +261,12 @@ function validateOptionsForMultiChunkOutput(outputOptions: NormalizedOutputOptio
 			code: 'INVALID_OPTION',
 			message: '"output.sourcemapFile" is only supported for single-file builds.'
 		});
+	if (!outputOptions.amd.autoId && outputOptions.amd.id)
```

Comment:
So following your comment, one last change suggestion (I would really do this myself but you know I cannot push...). As I want to release this right away and as you argued this could be breaking, we should make this a warning instead. To do so, you could either pass `this.inputOptions` or `this.inputOptions.onwarn` to this plugin and use `onwarn` instead of `error`.

Reply:
Done! I left the code as `INVALID_OPTION`. Not sure if that's correct or not for a warning.

>I would really do this myself but you know I cannot push...

I think you could change the base branch on the PR, so it would be possible to merge to a temp branch and then tweak that, but yeh it's not as straightforward